src/engine/blocks.js

```diff
@@ -564,7 +582,8 @@ class Blocks {
                 // Changing the value in a dropdown
                 block.fields[args.name].value = args.value;
 
-                if (!optRuntime){
+                if (!this.runtime){
+                    log.warn('Runtime is not optional, it should get passed in when the block container is created.');
```

Comment:
I think this check should be farther up (and fatal) or can just be removed altogether. We assume it exists above... and all over the place elsewhere.

Reply:
@rschamp, I think you're right. We had this check from before updating tests to resolve #1942 and this can probably be updated now.